queries.sql

```diff
@@ -1,10 +1,23 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT NAME from animals WHERE name LIKE '%mon';
 SELECT NAME from animals WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-01-01';
 SELECT NAME from animals WHERE NEUTERED = true AND ESCAPE_ATTEMPTS < 3;
 SELECT DATE_OF_BIRTH from animals WHERE NAME IN ('Agumon', 'Pikachu');
 SELECT (NAME, ESCAPE_ATTEMPTS) from animals WHERE WEIGHT_KG > 10.50;
 SELECT NAME from animals WHERE NEUTERED = true;
 SELECT NAME from animals WHERE NAME != 'Gabumon';
 SELECT NAME from animals WHERE WEIGHT_KG >=10.4 AND WEIGHT_KG <= 17.3;
+
```

Comment:
- [ ] Before the delete transaction you have to add transactions as given in the project requirements. Make sure to add all transactions and remember to begin it with `Begin` and end with `Commit`.
<img width="744" alt="Screenshot 2022-10-18 at 7 09 30 PM" src="https://user-images.githubusercontent.com/70269507/196484907-61d57a27-1226-411a-822c-2e2fde59e227.png">

Reply:
Thank you, I really missed something and now corrected them. Thanks once again.